javacutil/src/main/java/org/checkerframework/javacutil/TreePathUtil.java

```diff
@@ -467,4 +467,26 @@ public static String leafToStringTruncated(@Nullable TreePath path, int length)
     }
     return TreeUtils.toStringTruncated(path.getLeaf(), length);
   }
+
+  /**
+   * Retrieves the nearest enclosing method or class element for the specified path in the AST. This
+   * utility method prioritizes method elements over class elements. It returns the element of the
+   * closest method scope if available; otherwise, it defaults to the enclosing class scope. This
+   * method replaces the CFGTranslationPhaseOne#findOwner method.
```

Comment:
Can remove the last sentence here, since that method no longer exists and was private before anyway